manifests/m/Microsoft/ServiceFabricSDK/7.0.1949/Microsoft.ServiceFabricSDK.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=MDSU.CRLF.7-3-7.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.ServiceFabricSDK
+PackageVersion: 7.0.1949
+InstallerType: msi
+InstallerSwitches:
+  Silent: /quiet /qn
+  Custom: /norestart
+UpgradeBehavior: install
```

Comment:
```suggestion
UpgradeBehavior: install
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.ServiceFabricRuntime
```

Reply:
Dependencies key exists on line 18. Is this suggestion to only alter the order of the YAML keys ?

Reply:
I'm sorry, I missed it.